src/components/inspectors/TimerExpression.vue

```diff
@@ -168,15 +168,18 @@ export default {
       return 'You must select at least one day.';
     },
     iso8606Expression() {
-      if (this.selectedWeekdays.length === 1 && this.sameDay) {
+      if ((this.selectedWeekdays.length === 1 && this.sameDay) || this.periodicity !== 'week') {
```

Comment:
Could you extract this into a method? It would make the intent of this code much clearer.